### PR TITLE
R4 RelatedPerson Patient level patch enhancement

### PR DESCRIPTION
**Pull request by AnanyaRamesh14**
Description
----
Changes for RelatedPerson patient level patch Implementation
image

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/base/individuals/related-person.md [hunk 1]
```diff
@@ -214,15 +214,14 @@ The common [errors] and [OperationOutcomes] may be returned.
 
 ## Patch
 
-Patch an existing Encounter-level RelatedPerson.
+Patch an existing Encounter-level or Person-level RelatedPerson.
```
Comment (mani7002) [inline content/millennium/r4/base/individuals/related-person.md:217]: https://github.com/cerner/fhir.cerner.com/pull/686/files#diff-2e663b902e2663663b5f937e76fc0abb32406c03fbea8ee6d4dc4a7f6eb56584L225 here the line you deleted has patient level and we have named it as person level is it fine @SantanuBanerjee

Comment (AnanyaRamesh14) [reply content/millennium/r4/base/individuals/related-person.md:217]: https://jira2.cerner.com/browse/OPENSVC-20543
have made the changes as per mentioned in the jira description by Alex

Comment (SantanuBanerjee) [reply content/millennium/r4/base/individuals/related-person.md:217]: I think it is fine since this note was added in the jira by Alex.

**Review by SantanuBanerjee — APPROVED** [review]

**Review by mani7002 — APPROVED** [review]

**Review by Nandita92 — APPROVED** [review]

File: content/millennium/r4/base/individuals/related-person.md (hunk 1 above)
Comment (alexbeckner1) [inline content/millennium/r4/base/individuals/related-person.md:217]: Person-level should change to Patient-level. I apologize for having that incorrect on the JIRA. I went ahead and updated that on the JIRA to Patient-level.

Comment (AnanyaRamesh14) [reply content/millennium/r4/base/individuals/related-person.md]: updated here https://github.com/cerner/fhir.cerner.com/pull/686/commits/31e9db51448df7fb3b00319bcefbface2ce5343c#diff-2e663b902e2663663b5f937e76fc0abb32406c03fbea8ee6d4dc4a7f6eb56584R217

Comment (alexbeckner1) [reply content/millennium/r4/base/individuals/related-person.md]: Thank you.

**Review by alexbeckner1 — APPROVED** [review]

**Review by exopsRamba — APPROVED** [review]

File: content/millennium/r4/base/individuals/related-person.md [hunk 2]
```diff
@@ -214,15 +214,14 @@ The common [errors] and [OperationOutcomes] may be returned.
```
Comment (sydneanschutz) [inline content/millennium/r4/base/individuals/related-person.md:214]: Under create is says 'Create a new Encounter-level RelatedPerson.' but they can create a patient level too right? should we update that as well?

Comment (AnanyaRamesh14) [reply content/millennium/r4/base/individuals/related-person.md:214]: There is a separate PR for Create changes that will be done in the coming days

**Review by sydneanschutz — APPROVED** [review]

**Pull request merged**